source/agora/consensus/state/ValidatorSet.d

```diff
@@ -236,18 +199,20 @@ public class ValidatorSet
 
         Params:
             enroll_hash = key for an enrollment block height
+            height = height to get enrollment
 
         Returns:
             the enrolled block height, or `ulong.max` if no matching key exists
 
     ***************************************************************************/
 
-    public Height getEnrolledHeight (in Hash enroll_hash) @trusted nothrow
+    public Height getEnrolledHeight (in Height height, in Hash enroll_hash) @trusted nothrow
     {
         try
         {
-            auto results = this.db.execute("SELECT enrolled_height FROM validator_set" ~
-                " WHERE key = ? AND active = ?", enroll_hash, EnrollmentStatus.Active);
+            auto results = this.db.execute("SELECT enrolled_height FROM validator_set " ~
+                "WHERE key = ? AND enrolled_height <= ? AND enrolled_height >= ?",
```

Comment:
IDK if it's just me but I'd find the condition more readable if the condition was inverted, with `>=` then `<=`. Because you should give the lower bound before the upper one. But not a big deal either way.

Reply:
Yes I think I agree with you. I think it was just because I added the second check later. I will swap them over.